cmd/secrets-store-csi-driver/main.go

```diff
@@ -51,11 +52,12 @@ import (
 )
 
 var (
-	endpoint           = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	driverName         = flag.String("drivername", "secrets-store.csi.k8s.io", "name of the driver")
-	nodeID             = flag.String("nodeid", "", "node id")
-	logFormatJSON      = flag.Bool("log-format-json", false, "set log formatter to json")
-	providerVolumePath = flag.String("provider-volume", "/etc/kubernetes/secrets-store-csi-providers", "Volume path for provider")
+	endpoint                = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
+	driverName              = flag.String("drivername", "secrets-store.csi.k8s.io", "name of the driver")
+	nodeID                  = flag.String("nodeid", "", "node id")
+	logFormatJSON           = flag.Bool("log-format-json", false, "set log formatter to json")
+	providerVolumePath      = flag.String("provider-volume", "/etc/kubernetes/secrets-store-csi-providers", "Volume path for provider")
```

Comment:
Should we change the default to `/var/run` and move `/etc/kubernetes/*` to `additional-provider-volume-paths`?

Reply:
I kind of wanted this one to stay with the `/etc` default, next release we swap them, and release after that we make additional providers the empty string.

Reply:
Opened https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/870 to track this